packages/blade/src/components/Dropdown/_decisions/decisions.md

```diff
@@ -0,0 +1,672 @@
+   *
+   * @default single
+   */
+  variant?: 'single' | 'multiple';
+
+  /** Callback that gets called when item is checked/clicked */
+  onChange?: () => void;
+
+  /**
+   * Changes the background surface level color
+   *
+   * @default 2
+   */
+  backgroundLevel: 2 | 3;
+};
+```
+
+### `ActionList.Item`
+
+![](2022-11-29-17-29-01.png)
+
+#### Design
+
+- [Figma - ActionListItem / Body](https://www.figma.com/file/sAdplk2uYnI2ILnDKUxycW/Blade---Banking-Dark?node-id=13136%3A378703&t=RoEboUS3qWdStKK6-0)
+
+#### API
+
+Usage
+
+```jsx
+<ActionList.Item
+  title="Accept API"
+  value="accept"
+  desciption="Accept this API"
+  leading={<StarIcon />}
+  intent="negative"
+  trailing={<Text>⌘ + Space + A</Text>}
+  onClick={() => {
+    console.log('Item clicked');
+  }}
+/>
+```
+
+Props
+
+```ts
+type ActionListItemProps = {
+  title: string;
+
+  description: string;
+
+  /** Value that we use in JS form. Similar to `value` in `Checkbox` */
+  value: string;
+
+  /**
+   * Leading Icon or assets. Accepts JSX Component
+   */
+  leading?: React.ReactNode;
+
+  /**
+   * Trailing Assets. Should ideally be Text component for now
+   */
+  trailing?: React.ReactNode;
+
+  /**
+   * Changes item color to red
+   *
+   * @default neutral
+   */
+  intent?: 'negative' | 'neutral';
+
+  /** Click event on item */
+  onClick?: () => void;
+
+  /** URL. Should render item as `a` when this is passed */
+  href?: string;
+};
+```
+
+### `ActionList.Section`
+
+Just an API that wraps section inside ActionList.
+
+This entire component is optional. So it may not be added in the ActionList that doesn't have any sections.
+
+It should also add divider after all items.
+
+#### API
+
+```jsx
+<ActionList>
+  <ActionList.Section title="Section Heading">
+    <ActionList.Item
+    // ...props
+    />
+  </ActionList.Section>
+</ActionList>
+```
+
+<details>
+<summary>Previously Considered API</summary>
+
+```jsx
+<ActionList>
+  <ActionList.SectionHeading title="Section Heading" />
+  <ActionList.Item
+  // ...props
+  />
+  <ActionList.Divider />
+</ActionList>
+```
+
+We dropped this because people might tend to put Dividers after every item. It doesn't really explain grouping in any way whereas `ActionList.Section` API clearly defines the sections in items.
+
+</details>
+
+#### Props
+
+```ts
+type ActionListSectionProps = {
+  /** Section Heading. Should be optional */
+  title?: string;
+};
+```
+
+### `ActionList.Header`
+
+Header of the ActionList
+
+<img src="2022-11-29-17-36-50.png" alt="ActionList.Header Figma Design" width="300" />
+
+#### Design
+
+- [Figma - ActionListItem / Header](https://www.figma.com/file/sAdplk2uYnI2ILnDKUxycW/Blade---Banking-Dark?node-id=13136%3A378703&t=RoEboUS3qWdStKK6-0)
+
+#### API
+
+Usage
+
+```jsx
+<ActionList.Header title="Recent Searches" icon={HistoryIcon} />
+```
+
+Props
+
+```ts
+type ActionListHeaderProps = {
+  title: string;
+  icon: IconComponent;
```

Comment:
can we ask design if only these are 2 things that will ever exist in header?

Reply:
added it to open-questions